paprika/utils.py

```diff
@@ -11,6 +12,28 @@
 logger = logging.getLogger(__name__)
 
 
+def get_key(dct, value):
+    """
+    Get dictionary key given the value.
+
+    NOTE: this function will return a list of keys if there are more than one key with the same value in the order they are found.
+    """
+    key = [key for key in dct if (dct[key] == value)]
+
+    if len(key) > 1:
+        warnings.warn("There more than one key with the same value. Please check if this is the desired output.")
```

Comment:
`warnings.warn`? `logger.warning` is what I'd expect.

Reply:
Yes, I'll change this to `logger.warning`.